### PR TITLE
pro: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 27.14.4~22.04

### CODE REVIEW

File: pages/linux/pro.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pro
+
+> Manage Ubuntu Pro services.
+> More information: <https://ubuntu.com/pro>.
```
Comment (kbdharun) [inline pages/linux/pro.md:4]:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/latest/en/man1/ubuntu-advantage.1.html>.
```
I would suggest linking to the man page directly instead.

File: pages/linux/pro.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (kbdharun) [inline pages/linux/pro.md:5]:
```suggestion

- Connect your system to the Ubuntu Pro support contract:

`pro attach {{token}}`

```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/pro.md (hunk 2 above)
Comment (Rudloff) [reply pages/linux/pro.md:5]: The command needs to be run as root and IMHO using the interactive mode is more user friendly than passing a token:
```suggestion

- Connect your system to the Ubuntu Pro support contract:

`sudo pro attach`

```

File: pages/linux/pro.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+`pro security-status`
+
+- List packages that are no longer available for download:
+
+`pro security-status --unavailable`
+
+- List third-party packages:
+
+`ubuntu-security-status --thirdparty`
```
Comment (kbdharun) [inline pages/linux/pro.md:28]:
```suggestion
`pro ubuntu-security-status --thirdparty`
```
Shouldn't `pro` be present here too?

Comment (Rudloff) [reply pages/linux/pro.md:28]: Oh right, this is the correct command (it replaces `ubuntu-security-status`):
```suggestion
`pro security-status --thirdparty`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**